.mci.yml

```diff
@@ -278,6 +278,9 @@ functions:
                   set -o xtrace
 
                   if [ "Windows_NT" == "$OS" ]; then
+                      CRUD_TESTS_PATH=$(cygpath -m $CRUD_TESTS_PATH)
+                      ENCRYPTION_TESTS_PATH=$(cygpath -m $ENCRYPTION_TESTS_PATH)
```

Comment:
I don't think these paths are necessary. The `CRUD_TESTS_PATH` was here before, but I believe it was redundant. These tests are run through the RUN_TESTS target, and AFAICT they use the paths that are defined in src/mongocxx/test/CMakeLists.txt. Other evidence of that is that none of the other test paths are specified here (e.g. `TRANSACTIONS_TESTS_PATH`). Can you remove this?

Reply:
These are the lines that fixed my windows failure...

Reply:
Perhaps the CRUD path is unnecessary?  The ENCRYPTION redefine with cygpath here seems to be necessary to help windows find the right files in the test.

Reply:
Ah sorry, I don't think I fully understand how paths are getting through. If this is necessary for Evergreen hosts, then I'm fine with it. I would think the CRUD path is unnecessary, but if we remove the need for passing paths through environment variables in https://jira.mongodb.org/browse/CXX-2036, this will go away in the future anyway.

Reply:
Yes, I'm not 100% sure why it's necessary to call cygpath here, but not for all the test environment variables.  And since we are planning to remove these soon, I don't want to go too far down that rabbit hole!